packages/eslint-plugin-lit-a11y/lib/rules/alt-text.js

```diff
@@ -4,16 +4,19 @@
  */
 
 const { TemplateAnalyzer } = require('../../template-analyzer/template-analyzer.js');
+const { isHtmlTaggedTemplate } = require('../utils/isLitHtmlTemplate.js');
 
 //------------------------------------------------------------------------------
 // Rule Definition
 //------------------------------------------------------------------------------
 
-module.exports = {
+/** @type{import('eslint').Rule.RuleModule} */
+const AltTextRule = {
   meta: {
+    type: 'suggestion',
     docs: {
       description: 'Images require alt text',
```

Comment:
unrelated to this PR, should this rule allow `<img role="presentation"/>`?

Reply:
yep, would be a good addition 👍 maybe we can create a new hacktoberfest issue for it? Or we can also do it in this PR if you prefer

Reply:
spun out #1872